jujugui/static/gui/src/app/templates/storefront-search.handlebars

```diff
@@ -0,0 +1,96 @@
+<div class="row no-padding-top">
+  <div class="inner-wrapper list-block">
+    {{#if hasResults}}
+      {{#if text}}
+        <div class="twelve-col list-block__title no-margin-bottom">
+          Your search for &lsquo;{{text}}&rsquo; returned {{solutionsCount}} results.
+        </div>
+      {{/if}}
+      <div class="list-block__filters">
+        <nav class="six-col list-block__type">
+          <ul>
+            <li class="{{#compare currentType 'all'}}selected{{/compare}}" data-type="all" tabindex="0" role="button">All</li>
```

Comment:
The scope can probably be cut back a little bit here as we're just going to show a list of results for now.